roles/build_containers/tasks/main.yml

```diff
@@ -90,6 +90,14 @@
     - cifmw_build_containers_buildah_push | default ('false') | bool
     - not cifmw_build_containers_push_containers | bool
 
+- name: "Retag each image and push to registry: {{ item }}"
+  become: true
+  ansible.builtin.command: >
+    buildah tag {{ item }}:{{ cifmw_build_containers_image_tag }} {{ item }}:{{ cifmw_build_containers_retag_string }}
+    buildah push {{ item }}:{{ cifmw_build_containers_retag_string }} {{ item }}:{{ cifmw_build_containers_retag_string }}
```

Comment:
Based on my testing in different pr https://github.com/openstack-k8s-operators/ci-framework/pull/2996/files#diff-35c48fff63350c3d85b3c2c536183b5a12e5ad5d5858f74a5a9eecf3e2aa6d00R96
```suggestion
    buildah push --format v2s2 --all {{ item }}:{{ cifmw_build_containers_image_tag }} docker://{{ item }}:{{ cifmw_build_containers_retag_string }}
```
above command fixes my issue.
Below was the past issue 
![Screenshot From 2025-05-20 09-26-42](https://github.com/user-attachments/assets/efc55313-3790-4fdf-8ee0-850862bd494d)
and after running above command, it got addressed and now latest hash pointing to current .
![Screenshot From 2025-05-20 10-59-35](https://github.com/user-attachments/assets/79590c12-fd97-4ce9-b3af-f460af12d5db)

Reply:
nit: I suggest to add condintion:
```
when: cifmw_build_containers_tag_string != cifmw_build_containers_retag_string